[2022-07-05 07:21:28,402] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T19:00:00+00:00 [queued]>
[2022-07-05 07:21:28,467] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T19:00:00+00:00 [queued]>
[2022-07-05 07:21:28,467] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:21:28,468] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 07:21:28,468] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:21:28,514] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-04 19:00:00+00:00
[2022-07-05 07:21:28,545] {standard_task_runner.py:52} INFO - Started process 581 to run task
[2022-07-05 07:21:28,578] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-04T19:00:00+00:00', '--job-id', '248', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmph41yacj1', '--error-file', '/tmp/tmpzqsbetog']
[2022-07-05 07:21:28,579] {standard_task_runner.py:77} INFO - Job 248: Subtask kimuksung_test
[2022-07-05 07:21:28,964] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T19:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 07:21:29,398] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T19:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-04T19:00:00+00:00
[2022-07-05 07:21:29,405] {logging_mixin.py:109} INFO - test function start
[2022-07-05 07:21:29,405] {taskinstance.py:1718} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/test.py", line 21, in test
    parameter=context["paramas"]["parameter1"]
KeyError: 'paramas'
[2022-07-05 07:21:29,545] {taskinstance.py:1282} INFO - Marking task as FAILED. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220704T190000, start_date=20220705T072128, end_date=20220705T072129
[2022-07-05 07:21:29,682] {standard_task_runner.py:92} ERROR - Failed to execute job 248 for task kimuksung_test
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/test.py", line 21, in test
    parameter=context["paramas"]["parameter1"]
KeyError: 'paramas'
[2022-07-05 07:21:29,733] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-07-05 07:21:29,863] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-05 07:25:39,433] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T19:00:00+00:00 [queued]>
[2022-07-05 07:25:39,514] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T19:00:00+00:00 [queued]>
[2022-07-05 07:25:39,514] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:25:39,514] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 07:25:39,514] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:25:39,652] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-04 19:00:00+00:00
[2022-07-05 07:25:39,689] {standard_task_runner.py:52} INFO - Started process 771 to run task
[2022-07-05 07:25:39,706] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-04T19:00:00+00:00', '--job-id', '279', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp1buboqln', '--error-file', '/tmp/tmp4kmsh2g4']
[2022-07-05 07:25:39,707] {standard_task_runner.py:77} INFO - Job 279: Subtask kimuksung_test
[2022-07-05 07:25:40,138] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T19:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 07:25:40,368] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T19:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-04T19:00:00+00:00
[2022-07-05 07:25:40,369] {logging_mixin.py:109} INFO - test function start
[2022-07-05 07:25:40,369] {test.py:22} INFO - pubg
[2022-07-05 07:25:40,369] {python.py:175} INFO - Done. Returned value was: test function clear
[2022-07-05 07:25:40,436] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220704T190000, start_date=20220705T072539, end_date=20220705T072540
[2022-07-05 07:25:40,491] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 07:25:40,569] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-05 08:06:15,007] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T19:00:00+00:00 [queued]>
[2022-07-05 08:06:15,049] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T19:00:00+00:00 [queued]>
[2022-07-05 08:06:15,050] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:06:15,050] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 08:06:15,050] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:06:15,098] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-04 19:00:00+00:00
[2022-07-05 08:06:15,104] {standard_task_runner.py:52} INFO - Started process 2155 to run task
[2022-07-05 08:06:15,120] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-04T19:00:00+00:00', '--job-id', '330', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpgw25xf22', '--error-file', '/tmp/tmp10ky6sir']
[2022-07-05 08:06:15,121] {standard_task_runner.py:77} INFO - Job 330: Subtask kimuksung_test
[2022-07-05 08:06:15,292] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T19:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 08:06:15,509] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T19:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-04T19:00:00+00:00
[2022-07-05 08:06:15,510] {logging_mixin.py:109} INFO - test function start
[2022-07-05 08:06:15,511] {test.py:22} INFO - pubg
[2022-07-05 08:06:15,511] {python.py:175} INFO - Done. Returned value was: test function clear
[2022-07-05 08:06:15,606] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220704T190000, start_date=20220705T080615, end_date=20220705T080615
[2022-07-05 08:06:15,697] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 08:06:15,832] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-07-05 08:11:20,372] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T19:00:00+00:00 [queued]>
[2022-07-05 08:11:20,549] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T19:00:00+00:00 [queued]>
[2022-07-05 08:11:20,550] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:11:20,550] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 08:11:20,550] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:11:20,662] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-04 19:00:00+00:00
[2022-07-05 08:11:20,680] {standard_task_runner.py:52} INFO - Started process 2452 to run task
[2022-07-05 08:11:20,718] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-04T19:00:00+00:00', '--job-id', '393', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp0yuebuxa', '--error-file', '/tmp/tmptw9hsks5']
[2022-07-05 08:11:20,718] {standard_task_runner.py:77} INFO - Job 393: Subtask kimuksung_test
[2022-07-05 08:11:21,371] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T19:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 08:11:22,058] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T19:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-04T19:00:00+00:00
[2022-07-05 08:11:22,091] {logging_mixin.py:109} INFO - test function start
[2022-07-05 08:11:22,091] {test.py:22} INFO - pubg
[2022-07-05 08:11:22,091] {python.py:175} INFO - Done. Returned value was: pubg
[2022-07-05 08:11:22,311] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220704T190000, start_date=20220705T081120, end_date=20220705T081122
[2022-07-05 08:11:22,477] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 08:11:22,737] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
